File: JAVA/src/main/java/com/inf_112/trabalho_final/model/Cliente.java
package com.inf_112.trabalho_final.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class Cliente extends Pessoa {
    private double envergadura;
    private double peso;
    private double altura;
    private double torax;
    private double cintura;
}

File: APP/src/app/model/cliente.ts
import { Pessoa } from "./pessoa";

export class Cliente extends Pessoa {
    envergadura: number;
    peso: number;
    altura: number;
    torax: number;
    cintura: number;

    constructor() {
        super();
        this.envergadura = 0;
        this.peso = 0;
        this.altura = 0;
        this.torax = 0;
        this.cintura = 0;
    }
}
